packages/paste-core/components/in-page-navigation/src/OverflowButton.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, BoxProps, BoxStyleProps } from "@twilio-paste/box";
+import { ChevronLeftIcon } from "@twilio-paste/icons/esm/ChevronLeftIcon";
+import { ChevronRightIcon } from "@twilio-paste/icons/esm/ChevronRightIcon";
+import { BoxShadow } from "@twilio-paste/style-props";
+import { useTheme } from "@twilio-paste/theme";
+import React from "react";
+
+import { InPageNavigationContext } from "./InPageNavigationContext";
+
+interface OverflowButtonProps {
+  onClick: () => void;
+  position: "left" | "right";
+  visible?: boolean;
+  element?: BoxProps["element"];
+  showShadow?: boolean;
+}
+
+const Styles: BoxStyleProps = {
+  color: "colorTextIcon",
+  _hover: {
+    color: "colorTextPrimaryStrongest",
+    cursor: "pointer",
+  },
+};
+
+const InverseStyles: BoxStyleProps = {
+  color: "colorTextIconInverse",
+  _hover: {
+    color: "colorTextInverseWeak",
```

Comment:
```suggestion
    color: "colorTextInverse",
```

Needs to go to a brighter token to be 3:1 contrast with colorTextIconInverse